### ISSUE TITLE
Возможность скачивать треки вместе с текстом песни

### ISSUE THREAD

**Issue by MihailovAlexander** [egoroof/yandex-music-fisher#63]
Когда хочется попеть песни
И нету их текста 
ИЛИ PowerAMP не может подтянуть текст
ТОГДА было бы здорово гарантированно иметь их в *.mp3
Спасибо.

**Comment by Gravity89** [comment]
Когда хочется попить пива
И нет его рядом
ИЛИ друг не может подтянуть брикет
ТОГДА было бы здорово гарантированно иметь его в *.mp3

П.С. Простите, не удержался. 
Ну а по существу неплохо было бы все что Яндекс отдает заполнять в теги. Включая и текст песни. Гарантировать правда вам никто не станет. Даже Яндекс не имеет текста на каждую композицию.

**Comment by Zik** [comment]
При скачивании же и так заполняются все теги которые даёт Яндекс, в том числе текст песни :)
А вот если к примеру у песни нету текста и он подхватывал бы его к примеру из другой обширной базы, было бы здорово...но по моему это сложно :)

**Comment by dokxpi** [comment]
Не надо нам обширной  базы

**Comment by Zik** [comment]
Вот и я о том же :)

**Comment by Yadro** [comment]
> При скачивании же и так заполняются все теги которые даёт Яндекс, в том числе текст песни :)
Orly?

**Comment by dokxpi** [comment]
@Yadro  да, но это только с  ![Image of Yaktocat](https://github.com/egoroof/yandex-music-fisher/raw/master/readme_img/blue.png)
![Image of Yaktocat](https://github.com/egoroof/yandex-music-fisher/raw/master/readme_img/green.png) и ![Image of Yaktocat](https://github.com/egoroof/yandex-music-fisher/raw/master/readme_img/yellow.png) - идут лесом..!

@egoroof почему так? Можно исправить/поправить?

**Comment by egoroof** [comment]
@dokxpi потому, что яндекс отдаёт текст песни только при обращении к одиночному треку. А когда треки берутся пачкой - текста нет. Когда-нибудь может и поправлю

**Comment by Ilya911** [comment]
Очень жду данную возможность
